lib/python/qmk/commands.py

```diff
@@ -237,3 +238,71 @@ def parse_configurator_json(configurator_file):
             user_keymap['layout'] = aliases[orig_keyboard]['layouts'][user_keymap['layout']]
 
     return user_keymap
+
+
+def git_check_repo():
```

Comment:
Seems like we need a new file just for git operations 😄

Reply:
I was thinking the same thing, but I'm already working on a bunch of refactors for this develop cycle so this can fold up into that work.

Reply:
Yeah, in favour of doing that.